packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts

```diff
@@ -140,6 +141,11 @@ export class ImageEditPlugin implements ImageEditor, EditorPlugin {
             case 'keyDown':
                 this.keyDownHandler(this.editor, event);
                 break;
+            case 'contentChanged':
+                if (event.source === ChangeSource.Format && event.formatApiName == 'insertImage') {
```

Comment:
Can we also add the option to allow different format sources? I think InsertImage in OWA uses 'InsertImage' ChangeSource

Reply:
@JiuqingSong opinions on this?

Reply:
I think there is no need to add it to the ChangeSource file nor change current InsertImage API, I think adding an additional parameter to the class for additional ChangeSources to select image could enough to support the OWA Insert Image scenario.

But I will let Jason chime in.

Reply:
I had another idea, instead of additional parameter for ChangeSource we can pass a parameter for formatApiName, because to insert the image, we usually use the formatContentModel which triggers Format change source, then we use the formatApiName to identify if an image is inserted